cmd/ttn-lw-cli/internal/simulate/simulate_util.pb.go

```diff
@@ -0,0 +1,123 @@
+// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package simulate
+
+import (
+	"strings"
+	"time"
+
+	"github.com/gogo/protobuf/types"
+	"go.thethings.network/lorawan-stack/v3/pkg/band"
+	"go.thethings.network/lorawan-stack/v3/pkg/errors"
+	"go.thethings.network/lorawan-stack/v3/pkg/ttnpb"
+)
+
+func _processPaths(paths []string) map[string][]string {
+	if len(paths) == 0 {
+		return nil
+	}
+	pathMap := make(map[string][]string, len(paths))
+	for _, p := range paths {
+		if !strings.Contains(p, ".") {
+			pathMap[p] = nil
+			continue
+		}
+		parts := strings.SplitN(p, ".", 2)
+		h, t := parts[0], parts[1]
+		if val, ok := pathMap[h]; ok {
+			if val == nil {
+				continue
+			}
+			pathMap[h] = append(pathMap[h], t)
+		} else {
+			pathMap[h] = []string{t}
+		}
+	}
+
+	return pathMap
+}
+
+var (
```

Comment:
I think the errors and `SetDefaults` should sit in a `simulate_util.go`. Otherwise if we regenerate this file we will override these functions.

Reply:
Actuallly all the functions in this file are added manually, I'll rename it to `simulate.go`